packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.stories.tsx

```diff
@@ -0,0 +1,149 @@
+    platformId: 'GUARDIAN_WEB',
+    clientName: 'dcr',
+    referrerUrl:
+        'http://localhost:3000/politics/2020/jan/17/uk-rules-out-automatic-deportation-of-eu-citizens-verhofstadt-brexit',
+};
+
+const testTracking: TestTracking = {
+    campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
+    campaignId: 'remote_epic_test',
+    abTestName: 'remote_epic_test',
+    abTestVariant: 'api',
+    componentType: 'ACQUISITIONS_EPIC',
+    products: ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'],
+};
+
+export const tracking: Tracking = {
+    ...pageTracking,
+    ...testTracking,
+};
+
+export const ChoiceCardsBannerBlue = Template.bind({});
+ChoiceCardsBannerBlue.args = {
+    bannerId: 'choice-cards-banner-blue',
+    countryCode: 'GB',
+    content: {
+        mainContent: {
+            heading: <>Lend us a hand in 2023</>,
+            subheading: null,
+            paragraphs: [
+                <>
+                    Shareholders or billionaire owner, we report on world events with accuracy, free
+                    from political and commercial influence. And unlike many others, we’re committed
+                    to keeping our reporting open for all readers. Every contribution, however big
+                    or small, makes a difference.
+                </>,
+            ],
+            highlightedText: (
+                <>
+                    Support us from as little as £1. If you can, please consider supporting us with
+                    a regular amount each month. Thank you.
+                </>
+            ),
+            primaryCta: {
+                ctaText: 'Support us',
+                ctaUrl: 'https://support.theguardian.com/contribute',
+            },
+            secondaryCta: null,
+        },
+        mobileContent: {
+            heading: <>Lend us a hand in 2023</>,
+            subheading: null,
+            paragraphs: [
+                <>
+                    Shareholders or billionaire owner, we report on world events with accuracy, free
+                    from political and commercial influence. And unlike many others, we’re committed
+                    to keeping our reporting open for all readers. Every contribution, however big
+                    or small, makes a difference.
+                </>,
+            ],
+            highlightedText: (
+                <>
+                    Support us from as little as £1. If you can, please consider supporting us with
+                    a regular amount each month. Thank you.
+                </>
+            ),
+            primaryCta: {
+                ctaText: 'Support us',
+                ctaUrl: 'https://support.theguardian.com/contribute',
+            },
+            secondaryCta: null,
+        },
+    },
+    isSupporter: false,
+    tracking,
+    choiceCardAmounts: {
+        testName: 'Storybook_test',
+        variantName: 'Control',
+        amounts: {
+            ONE_OFF: {
+                amounts: [5, 10, 15, 20],
```

Comment:
Probably minor - worth removing amounts in array beyond 2/3 (depending on hideChooseYourAmount)?

Reply:
The amounts used by the banner in prod, will be coming from the configured amounts in the RRCP. The logic in the component (i.e. around whether to display choose your amount) will dictate what's shown in the UI but this is just an example of the data that could be fed to this component